lib/plugins/aws/package/compile/events/alb/lib/target.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = {
+  getTargetId(functionName) {
+    const { provisionedConcurrency } = this.serverless.service.functions[functionName];
```

Comment:
Recently on function object a `targetAlias` was introduced -> https://github.com/serverless/serverless/blob/024146885a913f545ebf8b0f5f6734b7650c64cc/lib/plugins/aws/package/compile/functions/index.js#L533

It's there, if specific alias instead of `$LATEST` needs to be pointed as trigger target.

Let's read it from there instead of reading `provisionedConcurrency` info

Reply:
Thanks for your feedback, I've made the change after rebasing on master and tested it.
There is still an issue with the CI that doesn't seem related to my changes tho.

Reply:
> There is still an issue with the CI that doesn't seem related to my changes tho.

It's fixed now in master, ensure to update your branch with it

Reply:
@medikoo I've rebased on master, now I got a `npm ERR! 429 Too Many Requests - GET https://registry.npmjs.org/@commitlint%2fcli`for the command `npm update --depth 3 --no-optional --no-save`

Reply:
It looks as temporary issue with npm. No other builds were affected. I think with next update it'll clean nicely